task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   name: fbc-related-image-check
 spec:
   description: >-
-    Checks whether all images referenced in file-based catalog (FBC) are valid.
+    Checks whether all images referenced in file-based catalog (FBC) are valid by using Skopeo to inspect manifest content.
```

Comment:
Suggestion: Checks the validity of all the images referenced in the file-based catalog (FBC) by using Skopeo to inspect manifest content.

BTW what do we mean by "inspect manifest content"? Depending on this we may have to update the description again.